### ISSUE TITLE
404

### ISSUE THREAD

**Issue by edison23** [OfflineIMAP/offlineimap.github.io#1]
http://offlineimap.org/howitworks.html

**Comment by nicolas33** [comment]
I couldn't find a reference to this page. Isn't it a bookmark relicate of the old website?

**Comment by edison23** [comment]
Don't know, I accessed the page few weeks ago and it worked. I came there via google, now can't find it again either. It was page with rather detailed description of how syncing works in offlineimap.

**Comment by nicolas33** [comment]
Yes, I know. It was left out while migrating to the new website. The original blog post is out there:
http://blog.ezyang.com/2012/08/how-offlineimap-works/

I didn't keep it on the move because I couldn't find the traces we have the rights to re-publish this writing.

**Comment by nicolas33** [comment]
@ezyang : you might be interested in this since it's your blog post.

If it's easy enough, we might rewrite your initial blog post in markdown and put it in the new community's website blog. Or, simply put a link to your blog at your own convenience.

**Comment by ezyang** [comment]
What's the new community website license? I'd probably be happy to relicense it for that.

**Comment by nicolas33** [comment]
Oh, good question! I didn't think about that.
- The style is derivated from a basic jekyll (or github?) style I don't know the licence. I'll try to find this out while I already tried to find this out without success.
- Documentation, well never had licence (outside those generated from the source code which is clearly GPLv2).
- Currently, the usual basic copyright is applied to the blog posts, too. You might put your own work on any licence you'd like, though.

It's kinda hard if simply not impossible to put the same licence everywhere since the website was built by aggregating hunks from different sources.

**Comment by ezyang** [comment]
OK, I'll just CC BY-SA 3.0 it.

**Comment by thekix** [comment]
Hello,

I would like close this patch and, if is possible, create the howitworks.html page again.

What do you thing about transform the http://blog.ezyang.com/2012/08/how-offlineimap-works/ @ezyang 's page in markdown, include the author name in the top ("Author: Edward Z. Yang") of the page and include the "This pages is under license CC BY-SA 3.0".

@ezyang , @nicolas33 what do you think?

Best Regards,
kix

**Comment by nicolas33** [comment]
Sure, this is a good idea. We have the rights now thanks to @ezyang .
Feel free.
A link to the original page would be great IMHO. ;-)

**Comment by chris001** [comment]
> Sure, this is a good idea. We have the rights now thanks to [@ezyang](https://github.com/ezyang) . Feel free. A link to the original page would be great IMHO. ;-)

Blog post created, containing markdown syntax, with license mentioned, with link back to @ezyang original blog post.
